src/reducer/cart-reducer.js

```diff
@@ -0,0 +1,11 @@
+export const cartReducer = (state, action) => {
+  switch (action.type) {
+    case "set_to_cart":
+      return { ...state, cartData: action.payload };
+    case "set_qtty":
+      return {
+        ...state,
+        cartData: action.payload,
```

Comment:
As both cases are doing the same thing, You can combine them by giving a common type to both cases like `CHANGE_CART`. Also, add the default block to the switch statement.

Reply:
Done , Thanks for pointing those things out